app/controllers/calendar_controller.rb

```diff
@@ -20,45 +22,20 @@ def index
   private
 
   def create_calendar_props
-    @props = {conferenceRooms: @conference_rooms, events: @events,
-              days: @days, times: @times,
-              unitEventLengthInSeconds: EventGrouper::GRANULARITY}
+    @props = {conferenceRooms: @conference_rooms,
```

Comment:
Can you think of any way to have less instance variables in controller actions now? Ideally aim for having as many ivars as you use on your view (and not more than 3 :D).
 For example `@conference_rooms` doesn't need to be an instance variable.